000-vip-init.php

```diff
@@ -4,6 +4,10 @@
 	require __DIR__ . '/.secrets/vip-secrets.php';
 }
 
+if ( ! defined( 'A8C_PROXIED_REQUEST' ) ) {
+	define( 'A8C_PROXIED_REQUEST', false );
```

Comment:
@nickdaugherty Gooood question. In production, it is always defined in `vip-secrets.php`, so I think you'd generate a fatal if you try to set it in a client repo.